### ISSUE TITLE
[Development] Connect HLR Form with our Back-End Endpoints

### ISSUE THREAD

**Issue by YanaRoy** [department-of-veterans-affairs/va.gov-team#1933]
## User Story
As a veteran, I want to be able to submit a Higher Level Review so I can appeal a decision on one of my benefit compensations.

As a developer, I need to make all necessary connections between the Higher-level Review form and its endpoints are made so I can ensure the form is functioning.

## Goal
_Tie front end to API_

## Acceptance Criteria
- _Fully integrated HLR form_

## Tasks
- [ ] Code up the FE to BE
- [ ] Add end-to-end tests
- [ ] _ *This relates to epic ticket #1053 *Check the box below when this has been completed and comment "done"._

**Comment by Mottie** [comment]
If the address, email or phone number are updated by the veteran, the front-end is responsible to post these changes to the vets profile API.

Ref: https://dsva.slack.com/archives/CKUCFQAHJ/p1579807789010000?thread_ts=1579806073.001600&cid=CKUCFQAHJ

**Comment by luke-gcio** [comment]
Was blocked by broken contestable issues endpoint, hence the sprint 23 move to sprint 24.

**Comment by Mottie** [comment]
Implemention of the frontend code has been merged even though we still have some errors popping up while using the endpoint. These should resolve soon.

**Comment by luke-gcio** [comment]
Confirmed we did the original intent of the ticket - any future work will be carved off post-integration testing.
